client/src/components/home/subpages/Pricing.js

```diff
@@ -9,8 +9,8 @@ import { Check } from '@material-ui/icons';
 const CheckMark = styled(Check)`
   color: ${props => (props.basic ? '#1b1b1b' : '#5f29ff')};
   margin-right: 4px;
-  padding-bottom: 3px;
-  font-size: 18px !important;
+  padding-bottom: 2px;
+  font-size: 22px !important;
```

Comment:
Is there a better way to override styles than using the important flag? [https://material-ui.com/customization/overrides/](url)